iotilecore/test/test_debug/test_flash_board_step.py

```diff
@@ -0,0 +1,8 @@
+from iotile.core.hw.debug.flash_board_step import FlashBoardStep
+
+def test_flash_board_step():
+    args = {
+        'file': 'test_file',
+        'port': 'jlink:device=nrf52;serial=50000797'
+    }
+    step = FlashBoardStep(args)
```

Comment:
You should add some asserts to make this test meaningful.  Like making sure exceptions are raised when you don't give it the right info:

```
    with pytest.raises(ArgumentError):
         create_step_with_issues()
```